packages/rest-crud/README.md

```diff
@@ -0,0 +1,65 @@
+# @loopback/rest-crud
+
+REST API controller implementing default CRUD semantics.
+
+## Overview
+
+This module allows applications to quickly expose a model via REST API without
+having to implement a custom controller class.
+
+## Installation
+
+```sh
+npm install --save @loopback/rest-crud
+```
+
+## Basic use
+
+1. Define your model class, e.g. using `lb4 model` tool.
+
+2. Create a Repository class, e.g. using `lb4 repository` tool.
```

Comment:
Is it required? IIUC, you want to skip Repository class so that a model can be used to expose REST CRUD APIs with a datasource.

Reply:
A custom repository class is not required, this is just to make the documentation easier to follow.

This new component is meant to be a low-level building block to support https://github.com/strongloop/loopback-next/issues/2036, I am expecting that most people most people will use #2036 and don't leverage the current API directly.

For the initial docs, I choose a scenario that's easy to understand (because it's using existing concepts) and that's matching the scenario used for testing.

Depending on the outcome of the next tasks (especially the spike #2738), we may add more "meat" to this component and update or even rework the README along the way.